appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/interceptors/InterceptorManager.java

```diff
@@ -270,11 +271,11 @@ public void registerRuntimeInterceptor(Object o) {
     }
 
     public InterceptorManager.InterceptorChain getAroundTimeoutChain(
-            MethodDescriptor mDesc, Method beanMethod) {
+            MethodDescriptor mDesc, @SuppressWarnings("unused") Method beanMethod) {
```

Comment:
Generally I learned to avoid using this annotation with unused, because - the warning is true and valid, so it should be visible even if we don't plan to change it.

Reply:
I get that. I tend to suppress  it in files that are otherwise free of warnings to better see when a warning comes up but for a team with mixed IDEs this might not make as much sense as just leaving the warning in.